src/test/app/ValidatorList_test.cpp

```diff
@@ -733,13 +748,72 @@ class ValidatorList_test : public beast::unit_test::suite
             trustedKeys->onConsensusStart (activeValidators);
             BEAST_EXPECT(! trustedKeys->trusted (list[0]));
         }
+        {
+            // Test 1-9 configured validators
+            auto trustedKeys = std::make_unique <ValidatorList> (
+                manifests, manifests, env.timeKeeper(), beast::Journal ());
+
+            std::vector<std::string> cfgPublishers;
+            hash_set<PublicKey> activeValidators;
+
+            std::vector<std::string> cfgKeys;
+            cfgKeys.reserve(9);
+
+            while (cfgKeys.size() < cfgKeys.capacity())
+            {
+                auto const valKey = randomNode();
+                cfgKeys.push_back (toBase58(
+                    TokenType::TOKEN_NODE_PUBLIC, valKey));
+                activeValidators.emplace (valKey);
+
+                BEAST_EXPECT(trustedKeys->load (
+                    emptyLocalKey, cfgKeys, cfgPublishers));
+                trustedKeys->onConsensusStart (activeValidators);
+                BEAST_EXPECT(trustedKeys->quorum () ==
+                    ValidatorList::calculateQuorum(cfgKeys.size()));
+                for (auto const& key : activeValidators)
+                    BEAST_EXPECT(trustedKeys->trusted (key));
+            }
+        }
+        {
+            // Test 2-9 configured validators as validator
+            auto trustedKeys = std::make_unique <ValidatorList> (
+                manifests, manifests, env.timeKeeper(), beast::Journal ());
+
+            auto const localKey = randomNode();
+            std::vector<std::string> cfgPublishers;
+            hash_set<PublicKey> activeValidators;
+
+            std::vector<std::string> cfgKeys {
+                toBase58(TokenType::TOKEN_NODE_PUBLIC, localKey)};
+            cfgKeys.reserve(9);
+
+            constexpr std::array<std::size_t, 10> quorums = { 0, 1, 2, 3, 3, 4, 4, 5, 6, 7 };
```

Comment:
Consider a comment here too indicating that this is the +1 buffer for unlisted local key?